packages/middleware-sdk-transcribe-streaming/src/websocket-handler.ts

```diff
@@ -22,98 +22,144 @@ export class WebSocketHandler implements HttpHandler {
     handlerProtocol: "websocket",
   };
   private readonly connectionTimeout: number;
+  private readonly sockets: Record<string, WebSocket[]>;
+
   constructor({ connectionTimeout }: WebSocketHandlerOptions = {}) {
     this.connectionTimeout = connectionTimeout || 2000;
+    this.sockets = {};
```

Comment:
optional minor issue: this can be inline with the declaration, but leave it how you prefer.